### ISSUE TITLE
Target latest lts .net core

### ISSUE THREAD

**Issue by dv00d00** [ffMathy/FluffySpoon.AspNet.LetsEncrypt#32]
I'd like to have a NuGet package targetting .net core 3.1
Let's discuss how we can achieve it.

We can do something similar to what peeps in https://github.com/justeat/JustEat.StatsD/blob/master/src/JustEat.StatsD/JustEat.StatsD.csproj did

E.g. target multiple frameworks + msbuild voodoo for determining depending asp.net core nugets versions

Also, it is unclear how current artifacts are being published. Is it possible to move build pipeline to something like TravisCI or similar?

**Comment by ffMathy** [comment]
The pipeline will be moved to GitHub actions soon.

As for compatibility, we have to stick to .NET Standard in my opinion.

What features do you need from the latest .net core that aren't in .net standard?

**Comment by dv00d00** [comment]
Yeah, for some reason I thought nuget was targeting netcoreapp2.2, netstandard20 is everything that is needed atm.

I was woried about referenced nuget packages version ( e.g. Microsoft.Extensions.* ) all targeted 2.2.0. But then I found [this amazing article](https://andrewlock.net/converting-a-netstandard-2-library-to-netcore-3/). 

TLDR:

```
When used in an ASP.NET Core 2.x app, the 2.x.x libraries will be used, just as before. However, when you reference your library in an ASP.NET Core 3.0, the 2.x dependencies of your library will be automatically upgraded to the 3.0.0 versions due to the NuGet package resolution rules.
```

There might be a point when we would want a .net core 3.1 specific features though, then multi-targeting would work fine. No need in it atm
